src/dispatch/organization/models.py

```diff
@@ -0,0 +1,43 @@
+from typing import List, Optional
+
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.orm import relationship
+from sqlalchemy_utils import TSVectorType
+
+from dispatch.database import Base
+from dispatch.models import DispatchBase
+
+
+class Organization(Base):
```

Comment:
Is the `ContactMixin` model missing here?

Reply:
I don't think so.

Reply:
Ok, the `get_by_email` in the views module led me to believe that it was perhaps missing.